integration-tests/suites/base.go

```diff
@@ -434,8 +434,10 @@ func (s *IntegrationTestSuiteBase) getIPAddress(containerName string) (string, e
 	return s.Executor().GetContainerIP(containerName)
 }
 
-func (s *IntegrationTestSuiteBase) getPort(containerName string) (string, error) {
-	return s.Executor().GetContainerPort(containerName)
+func (s *IntegrationTestSuiteBase) getPort(containerName string) (uint32, error) {
```

Comment:
Using `uint16` for ports makes more sense than `uint32`, but `sensor.NetworkAddress` uses `uint32` for the Port field, so I am using `uint32` for ports.